api/controllers/shabads.js

```diff
@@ -600,3 +601,23 @@ const getNavigation = async (req, res, type, first, last, source = '') => {
   }
   return {};
 };
+
+exports.updates = async (req, res) => {
+  let conn;
+  const { Date } = req.params;
+
+  try {
+    conn = await req.app.locals.pool.getConnection();
+
+    if (!Date) res.json({ error: 'please enter valid date' });
+    const query = `SELECT COUNT(*) FROM Verse WHERE Updated >= '${Date}'`;
```

Comment:
Should probably use a parameterized query and/or do more thorough input sanitization on line 612

Reply:
Some input sanitization that need to be done
- Date in the future 
- Date not formatted properly. 

both of those should return 422 with appropriate messages.

Reply:
How do i check if the date is formatted correctly? Do i have to use some regex thingy or something else?

Reply:
Regex could work. check out https://regexlib.com/REDetails.aspx?regexp_id=95